docs/usage/configuration-options.md

```diff
@@ -2080,6 +2080,11 @@ Here's an example of how you would define PR priority so that devDependencies ar
 
 The PR title is important for some of Renovate's matching algorithms (e.g. determining whether to recreate a PR or not) so ideally don't modify it much.
 
+## pruneBranchAfterAutomerge
+
+Configure to `false` to disable puring of merged branches after automerge.
+Defaults to `true`.
```

Comment:
```suggestion
By default Renovate deletes, or "prunes", the branch after automerging.
Set `pruneBranchAfterAutomerge` to `false` to keep the branch after automerging.
```

How about something like this?

Reply:
Referred to an existing config for keeping things consistent, the suggested one looks more verbose and understandable. Committing the same.